django/db/models/fields/files.py

```diff
@@ -312,6 +314,15 @@ def get_prep_value(self, value):
 
     def pre_save(self, model_instance, add):
         file = super().pre_save(model_instance, add)
+        if file.name is None and getattr(file, "_file") is not None:
+            exc = FieldError(
+                f"File for {self.name} must have "
+                "the name attribute specified to be saved."
+            )
+            if PY311 and isinstance(getattr(file, "_file"), ContentFile):
```

Comment:
There is no need to use `getattr()`, without a default it’s the same as a regular attribute access.

```suggestion
        if file.name is None and file._file is not None:
            exc = FieldError(
                f"File for {self.name} must have "
                "the name attribute specified to be saved."
            )
            if PY311 and isinstance(file._file), ContentFile):
```